.circleci/config.yml

```diff
@@ -174,24 +174,12 @@ workflows:
       - docs:
           requires:
             - build
-      - docs-master:
-          filters:
-            branches:
-              only: master
-          requires:
-            - build
       - deploy:
           requires:
             - lint
             - test
             - storybook
             - docs
-      - deploy-master:
```

Comment:
Should you also delete https://github.com/adobe-private/react-spectrum-v3/blob/cf5410adaedecec6a6dacfe1b7b7f556973bbbb2/.circleci/config.yml#L135-L142 ?

Reply:
not really any reason to